### PR TITLE
docs: add t2 leadership recommendations file

### LINKED ISSUE #89: Leadership Recommendations for T2
Onboarding form improvements:
- skill confidence (1-5) eg. programming, troubleshooting, documentation, project collaboration etc
- do you want to work towards leadership?

- Product Lead + Area lead merged
- Addition of more area leads
- QA team
- Documentation team part of Internal Systems
- Delivery lead training in week 0
- Implement recommendations from T1 Delivery leads

### PR DESCRIPTION
**Pull request by ginge270**
# Description

This file outlines the recommendations from T1 Delivery leads about their project teams, and general recommendations from the T1 Area lead team

Fixes # (issue)

## Type of change

- [x] Documentation (update or new)

# How Has This Been Tested?

N/A

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Comment by ghost** [comment]
@thoth-tech/senior-leads is this completed? are ready for review?

File: docs/projects/t1-2022/thoth-tech-leadership/t2-leadership-project-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### Extensions
+
+- Touch base with Andrew to get more insights on the deliverables
+- Set up regular cadence with Andrew to surface any blockers early on
+- Require better documentation to help understand what is done and what is not yet completed
+- Leadership
+  - Jack and Ben are good, highly engaged, and self-started. They help to lead retrospective.
+
+## Modules
```
Comment (ghost) [inline docs/projects/t1-2022/thoth-tech-leadership/t2-leadership-project-recommendations.md]:
```suggestion
### Modules
```

File: docs/projects/t1-2022/thoth-tech-leadership/t2-leadership-project-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### Documentation
+
+- Move toward a Document Control role, having ownership over documentation and product documents.
+- Size: 6-10 students, sub teams for each product
+- Skills: good understanding of English language, understanding of markdown, strong engagement,
+  strong communication, and experience with GitHub
+- Good for people wanting to avoid coding, but still technical understanding to approve work.
+
+## SplashKit
```
Comment (ghost) [inline docs/projects/t1-2022/thoth-tech-leadership/t2-leadership-project-recommendations.md:81]: Just missing Applications
Operations is complete as of this tri (new epic created)

**Comment by ghost** [comment]
I am confused why this is failing now? Could someone please take a look at it?

**Comment by tancnle** [comment]
> I am confused why this is failing now? Could someone please take a look at it?

The new listing rules PR has been [merged](https://github.com/thoth-tech/handbook/pull/101) so we have to rebase and fix these issues before merging. Let me patch it up.

**Review by kats8 — APPROVED** [review]
Ready for merging.

**Pull request merged**